README.md

```diff
@@ -12,4 +12,49 @@
   reasons why this could be the case, given that reasoning with the asymptotic
   complexity suggests a different time.
 
+       
 Add your answers to this markdown file.
+
+- List 3 reasons why asymptotic analysis may be misleading with respect to
+  actual performance in practice.
+
+      1. Asymptotic comparisons can be misleading because dependent on the size of the array, the cases may differ. 
```

Comment:
The cases don't depend on the input size. The input size is explicitly part of every case through the $n$ in the asymptotic expression.